src/main/java/org/generationcp/breeding/manager/listmanager/GermplasmSearchResultsComponent.java

```diff
@@ -260,11 +261,20 @@ public String generateDescription(final Component source, final Object itemId, f
 				// set the default value to empty string instead of null for germplasm without name
 				String germplasmNames = "";
 				if (propertyId == GermplasmSearchResultsComponent.NAMES) {
-					final Item item = GermplasmSearchResultsComponent.this.matchingGermplasmTable.getItem(itemId);
-					final Integer gid =
-							Integer.valueOf(((Button) item.getItemProperty(ColumnLabels.GID.getName()).getValue()).getCaption());
+					try{
+						final Item item = GermplasmSearchResultsComponent.this.matchingGermplasmTable.getItem(itemId);
+						final int gid =
+								Integer.parseInt(((Button) item.getItemProperty(ColumnLabels.GID.getName()).getValue()).getCaption());
+
+						germplasmNames = GermplasmSearchResultsComponent.this.getGermplasmNames(gid);
+					}catch(MiddlewareException ex){
+						if(ex.getMessage().contains("Problem building pedigree string for gid")){
+							MessageNotifier.showError(GermplasmSearchResultsComponent.this.getWindow(), "Error with Cross Expansion", "There is a data problem that prevents the generation of the cross expansion. Please contact your administrator");
```

Comment:
Exception  swallowing happening here. As discussed, this is not the best place to handle MiddlewareException.